packages/import-bundle/src/index.js

```diff
@@ -32,6 +32,7 @@ export async function importBundle(bundle, options = {}, powers = {}) {
     inescapableTransforms = [],
     inescapableGlobalProperties = {},
     expectedSha512 = undefined,
+    importHook = () => null,
```

Comment:
Should be okay to simply omit, and obviate a closure allocation when it’s unused.
```suggestion
    importHook = undefined,
```